pkg/utils/docker.go

```diff
@@ -0,0 +1,40 @@
+package utils
+
+import (
+	"fmt"
+
+	"github.com/Sirupsen/logrus"
+
+	"github.com/kubernetes-incubator/kompose/pkg/kobject"
+)
+
+func DockerBuildImage(name string, service kobject.ServiceConfig, composeFileDir string) string {
+	image := name
+	if service.Image != "" {
+		image = service.Image
+	}
+
+	cmd := NewCommand(fmt.Sprintf("docker build -t %s %s", image, service.Build))
```

Comment:
I also think that Docker API would be better. Docker has nice official library for - https://docs.docker.com/engine/api/getting-started/

Using `os/exec` should be last resort, only if there is no other way.